bodhi/server/models.py

```diff
@@ -1339,6 +1433,21 @@ class ModuleBuild(Build):
         'polymorphic_identity': ContentType.module,
     }
 
+    @property
+    def nvr_name(self):
+        """Return the RPM Name."""
```

Comment:
```
"""
Return the ModuleBuild's name.

Returns:
    str: The name of the module.
"""
```

(This is name, and not name:stream, right? If not, please amend the message above as appropriate.